src/index.js

```diff
@@ -113,6 +121,14 @@ class Urls {
   servicesManifest() {
     return `${this.rootUrl}/references/manifest.json`;
   }
+
+  /**
+   * Returns a link to the task group or the task in Task Inspector
+   */
+  taskInspector(taskGroupId, taskId) {
+    let taskLog = taskId ? `/tasks/${taskId}/runs/0/logs/public/logs/live.log` : '';
+    return `${this.rootUrl}/groups/${taskGroupId}${taskLog}`;
```

Comment:
Let's use `ui` method here too.

Reply:
Logically I see there being two steps. One step is to work out what the path inside the taskcluster-web app is for viewing a task, regardless of whether it is the `taskcluster.net` deployment or any other one, and the second step is to work out the full path is, based on that subpath, and the root url. The second step is implemented in `ui` function, with the intention that everything that needs to perform this logical step, uses this function. This means that if at some point we want to change the logic, we can change it in one place only. So even though it is trivial to implement directly, if the implementation changes, you would still have to change it in two places, and that could be easy to lose track of.